scripts/variantstore/wdl/GvsExtractAvroFilesForHail.wdl

```diff
@@ -109,9 +118,9 @@ workflow GvsExtractAvroFilesForHail {
 
     call GenerateHailScripts {
         input:
-            go_non_superpartitioned = ExtractFromNonSuperpartitionedTables.done,
+            go_non_superpartitioned = [ExtractFromFilterTables.done, ExtractFromSampleInfoTable.done],
```

Comment:
could `&&` these and not change the type of the callee, also it would be kind of intuitive that both of them need to be `done`.